wagtail/admin/templates/wagtailadmin/shared/dropdown/dropdown.html

```diff
@@ -10,13 +10,13 @@
     - `toggle_aria_label` (string?) - aria-label for the toggle button
     - `toggle_describedby` (string?) - aria-describedby for the toggle button
     - `toggle_classname` (string?) - additional toggle classes
-    - `toggle_tippy_offset` (string?) - Tippy.js offset prop
+    - `toggle_tooltip_offset` (string?) - Tippy.js offset prop
     - `hide_on_click` (boolean?) - Whether or not the tooltip should hide when clicking inside
     - `children` - Dropdown contents (`a` and `button` elements only)
 {% endcomment %}
 
 <div data-controller="w-dropdown" class="{% classnames 'w-dropdown' classname %}" {{ attrs }} {% if hide_on_click %}data-w-dropdown-hide-on-click-value="true"{% endif %}>
-    <button type="button" class="{% classnames 'w-dropdown__toggle' toggle_label|yesno:',w-dropdown__toggle--icon' toggle_classname %}" data-w-dropdown-target="toggle"{% if toggle_aria_label %} aria-label="{{ toggle_aria_label }}"{% endif %}{% if toggle_describedby %} aria-describedby="{{ toggle_describedby }}"{% endif %}{% if toggle_tippy_offset %} data-tippy-offset="{{ toggle_tippy_offset }}"{% endif %}>
+    <button type="button" class="{% classnames 'w-dropdown__toggle' toggle_label|yesno:',w-dropdown__toggle--icon' toggle_classname %}" data-w-dropdown-target="toggle"{% if toggle_aria_label %} aria-label="{{ toggle_aria_label }}"{% endif %}{% if toggle_describedby %} aria-describedby="{{ toggle_describedby }}"{% endif %}{% if toggle_tooltip_offset %} data-tippy-offset="{{ toggle_tooltip_offset }}"{% endif %}>
```

Comment:
These data attributes also need updating.

```suggestion
    <button type="button" class="{% classnames 'w-dropdown__toggle' toggle_label|yesno:',w-dropdown__toggle--icon' toggle_classname %}" data-w-dropdown-target="toggle"{% if toggle_aria_label %} aria-label="{{ toggle_aria_label }}"{% endif %}{% if toggle_describedby %} aria-describedby="{{ toggle_describedby }}"{% endif %}{% if toggle_tooltip_offset %} data-w-dropdown-offset-value="{{ toggle_tooltip_offset }}"{% endif %}>
```

Sorry but you will need to make a change to client/src/controllers/DropdownController.ts also to support the offset there.